pkg/adapter/adapter.go

```diff
@@ -7,7 +7,7 @@ import (
 const (
 	HarborVulnReportv1MimeType    = "application/vnd.scanner.adapter.vuln.report.harbor+json; version=1.0"
 	RawVulnReportMimeType         = "application/vnd.scanner.adapter.vuln.report.raw+json"
-	DockerImageMimeType           = "application/vnd.oci.image.manifest.v1+json"
+	DockerImageMimeType           = "application/vnd.oci.image.manifest.v2+json"
```

Comment:
Should this replace, or merely be additive? This is used to indicate to Harbor the types of artifacts the scanner can process, so it be both v1 and v2. Was there discussion about this specific for a replacement rather than an addition?

Reply:
This is a straight replacement. It offers backward compatibility per my discussions with Steven